packages/component-boilerplate/src/hooks/graphql/useBaseQuery.ts

```diff
@@ -85,12 +85,11 @@ export function useBaseQuery<T = any>(
   initialState: BaseQueryData<T>
 ): [BaseQueryData<T>, (variables: unknown) => unknown] {
   const client: GraphQLClient = useGraphQLClient();
-  const isMountedRef = useIsMountedRef();
+  // const isMountedRef = useIsMountedRef();
```

Comment:
Flagging commented out code in this file for deletion

Reply:
deleted!